MVVMRxSwiftDemo/Sources/Presentation/Modules/Login/LoginViewModel.swift

```diff
@@ -49,8 +49,10 @@ final class LoginViewModel: LoginViewModelProtocol {
                     .login(email: inputs.email, password: inputs.password)
                     .trackIndicator(indicator)
                     .trackError(error)
-                    .asCompletable()
-                    .andThen(Single.just(homeBuilder.build()))
+                    .materialize()
+            }
+            .compactMap {
+                $0.isCompleted ? homeBuilder.build() : nil
```

Comment:
We have `filter` operator, so I would prefer this:
```suggestion
            .filter { $0.isCompleted }
            .map { _ in homeBuilder.build() }
```